solidjs-tailwind/src/App.jsx

```diff
@@ -6,6 +6,10 @@ import SigninPage from './pages/Signin';
 import RedirectPage from './pages/Redirect';
```

Comment:
I think the solidjs files got here by accident? 🙂

Reply:
Apparently, I update the branch with the main branch.